Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -433,14 +442,75 @@ cancelSearchPopUp push state =
   , accessibility DISABLE
   ][PopUpModal.view (push <<< CancelSearchAction) (cancelAppConfig state)]
 
-chatView :: forall w. (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
-chatView push state =
-  PrestoAnim.animationSet [ translateYAnimFromTop $ translateFullYAnimWithDurationConfig 300 ] $
+rideInfoView :: forall w. (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
+rideInfoView push state =
   relativeLayout
   [ height MATCH_PARENT
   , width MATCH_PARENT
-  , accessibility if state.props.showCallPopUp then DISABLE_DESCENDANT else DISABLE
-  ][ ChatView.view (push <<< ChatViewActionController) (chatViewConfig state) ]
+  , accessibility if state.data.settingSideBar.opened /= SettingSideBar.CLOSED || state.props.currentStage == ChatWithDriver || state.props.isCancelRide || state.props.isLocationTracking || state.props.callSupportPopUp || state.props.cancelSearchCallDriver || state.props.showCallPopUp || state.props.emergencyHelpModal || state.props.showRateCard || state.props.bottomSheetState == STATE_EXPANDED || state.data.waitTimeInfo then DISABLE_DESCENDANT else DISABLE
+  , clickable if (os == "IOS" || state.props.bottomSheetState == STATE_EXPANDED) then true else false
+  , background if state.props.bottomSheetState == STATE_EXPANDED then Color.black400 else Color.transparent
+  , orientation VERTICAL
+  ][ (if state.data.currentSearchResultType == QUOTES || not state.data.config.feature.enableChat || not state.data.config.feature.enableSuggestions then 
```

Comment:
@PraveenGongada create a function for this inside where block in the same function.